ci/ci-iwyu.py

```
@@ -416,6 +417,128 @@ def check_iwyu_available() -> tuple[bool, str]:
+        "-j",
+        jobs,
+        *files,
+        "--",
+    ] + iwyu_tail
+
+    print(
+        f"Running clang-tool-chain-iwyu-tool against {len(files)} src/ files "
+        f"using compile DB: {compile_db}"
+    )
+    try:
+        result = subprocess.run(cmd)
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)